src/fides/api/alembic/migrations/versions/3304082a6cee_scylla_connection_type.py

```diff
@@ -0,0 +1,46 @@
+"""scylla_connection_type
+
+Revision ID: 3304082a6cee
+Revises: 5fe01e730171
+Create Date: 2024-06-03 19:54:20.907724
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "3304082a6cee"
+down_revision = "5fe01e730171"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # Add to ConnectionType enum
+    op.execute("alter type connectiontype rename to connectiontype_old")
+    op.execute(
+        "create type connectiontype as enum('mongodb', 'mysql', 'https', 'snowflake', 'redshift', 'mssql', 'mariadb', 'bigquery', 'saas', 'manual', 'manual_webhook', 'timescale', 'fides', 'sovrn', 'attentive', 'dynamodb', 'postgres', 'generic_consent_email', 'generic_erasure_email', 'scylla')"
+    )
+    op.execute(
+        (
+            "alter table connectionconfig alter column connection_type type connectiontype using "
+            "connection_type::text::connectiontype"
+        )
+    )
+    op.execute("drop type connectiontype_old")
```

Comment:
Currently following the pattern of renaming the old enum, creating a new enum with the new connection type, switching to the new enum and deleting the old enum.  Is there any continued risk with this type of pattern? I think this sort of migration would be quick given the likely table sizes.

We discussed getting rid of the enum altogether but I'm not tackling that here for time -

Reply:
Note to self to double check these enum values before merge -

Reply:
i don't think there's any risk beyond the extra bit of manual work required, especially for coordination when we've got multiple changes to this in-flight at the same time (as you mention - thanks for calling that out!)

but totally fine by me that you're not tackling getting rid of the enum here 👍